src/core/api/link_raw_api.cpp

```diff
@@ -38,26 +38,43 @@
 #include "openthread-instance.h"
 #include "common/debug.hpp"
 #include "common/logging.hpp"
+#include "mac/mac.hpp"
 
-#if OPENTHREAD_ENABLE_RAW_LINK_API
+#if OPENTHREAD_RAW || OPENTHREAD_ENABLE_RAW_LINK_API
 
 otError otLinkRawSetEnable(otInstance *aInstance, bool aEnabled)
 {
     otError error = OT_ERROR_NONE;
 
-    VerifyOrExit(!aInstance->mThreadNetif.IsUp(), error = OT_ERROR_INVALID_STATE);
-
     otLogInfoPlat(aInstance, "LinkRaw Enabled=%d", aEnabled ? 1 : 0);
+#if OPENTHREAD_RAW
+
+    if (aEnabled)
```

Comment:
In ncp's raw mode, link raw's functionality is always enabled. The `LinkRaw::mEnabled` is actually a flag of whether the link raw functionality is activated under the existing implementation, and it's not whether the radio layer is enabled or not. This PR doesn't change this design. Shall I change this in this PR?